optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -498,8 +498,8 @@ private void writeAverageCalculateCountPerSecondSummaryChart() {
         }
         XYPlot plot = createScalabilityPlot(seriesList,
                 "Problem scale", NumberFormat.getInstance(locale),
-                "Average calculate count per second", NumberFormat.getInstance(locale));
-        JFreeChart chart = new JFreeChart("Average calculate count summary (higher is better)",
+                "Average calculation count per second", NumberFormat.getInstance(locale));
+        JFreeChart chart = new JFreeChart("Average calculation count summary (higher is better)",
```

Comment:
Ahh, sorry, I forgot to include an important notice: this is for 6.x stream, the fix doesn't apply to master.